sklearnex/svm/nusvc.py

```diff
@@ -83,6 +83,8 @@ def __init__(
     def fit(self, X, y, sample_weight=None):
         if sklearn_check_version("1.2"):
             self._validate_params()
+        elif self.nu <= 0 or self.nu > 1:
+            raise ValueError("Margin error upper bound outside range: nu <= 0 or nu > 1")
```

Comment:
Doesn't it make sense move this part into dispatching check for the support.
I would prefer stock scikit-learn to handle error raising

Reply:
THIS PR IS JUST TO VERIFY AGAINST CI TO SEE IF WINDOWS SKLEARN 1.0, 1.1 SEG FAULTS CAN BE STOPPED ON PYTHON SIDE

Reply:
Thank you for copying the PR description. I read it before watching it. I think it makes sense to respond to comments a little more politely. This repo is public, it makes sense to leave a comment in a more appropriate manner.